tests/fixtures/dataframes.py

```diff
@@ -10,6 +10,11 @@ def categorical():
     )
 
 
+@pytest.fixture
```

Comment:
If this fixture is only used in the Test class, then I would prefer to move it there